src/notional/query.py

```diff
@@ -281,14 +289,19 @@ def filter(self, filter=None, **kwargs):
 
         if filter is None:
 
-            if "property" in kwargs:
-                filter = PropertyFilter.parse_obj(kwargs)
-            elif "timestamp" in kwargs and kwargs["timestamp"] == "created_time":
-                filter = CreatedTimeFilter.parse_obj(kwargs)
-            elif "timestamp" in kwargs and kwargs["timestamp"] == "last_edited_time":
-                filter = LastEditedTimeFilter.parse_obj(kwargs)
+            if isinstance(self.endpoint, SearchEndpoint):
```

Comment:
Bummer that we need to look into the `notion_client` module to figure this out...  I can't think of an alternative at the moment.

Reply:
Yeah, it's not so clean...